src/System.Runtime/tests/System/EnumTests.netcoreapp.cs

```diff
@@ -74,10 +75,14 @@ public static void Parse_Invalid_NetCoreApp11(Type enumType, string value, bool
 
         public static IEnumerable<object[]> UnsupportedEnumType_TestData()
         {
+#if netcoreapp
```

Comment:
The reason there isn't an issue number is:

1. The issue is essentially ".Net Native doesn't support RefEmit"

2. There are lot of if's guarding this same set of statics (already preexisting in EnumTests.cs) and if we're going to visit all of those, it makes more sense to do it as part of a more comprehensive refactoring that gets rid of those ifs and gates these statics using PlatformDetection.IsReflectionEmitSupported. It was just too much to pile on to this PR. Opened https://github.com/dotnet/corefx/issues/19998 to track.